include/openssl/ssl.h.in

```diff
@@ -428,6 +431,28 @@ typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
 
 #define SSL_OP_PREFER_NO_DHE_KEX                         SSL_OP_BIT(35)
 
+#ifndef OPENSSL_NO_ECH
+    /* Set this to tell client to emit greased ECH values */
```

Comment:
nit: indentation ?

Reply:
sure (though indentation in that file seems all over the place)